legion/engine/core/logging/logging.hpp

```diff
@@ -168,6 +168,29 @@ namespace fmt
         }
     };
 
+    template <>
+    struct fmt::formatter<legion::core::math::ivec3> {
+        char presentation = 'f';
+
+        constexpr auto parse(format_parse_context& ctx) {
+            auto it = ctx.begin(), end = ctx.end();
+            if (it != end && (*it == 'f' || *it == 'e')) presentation = *it++;
+
+            if (it != end && *it != '}')
+                throw format_error("invalid format");
+
+            return it;
+        }
+
+        template <typename FormatContext>
+        auto format(const legion::core::math::ivec3& p, FormatContext& ctx) {
+            return format_to(
+                ctx.out(),
+                presentation == 'f' ? "({:}, {:}, {:})" : "({:}, {:}, {:})",
+                p.x, p.y, p.z);
+        }
```

Comment:
ivec3s should not have format specifiers, what does it mean to have floating vs exponential representation for whole numbers ?